### ISSUE TITLE
[question] what is the meaning of pwin vector

### ISSUE THREAD

**Issue by genmeblog** [BonyJordan/twixtbot#2]
What do these numbers mean?

`pwin= [[ 1.4218217 -3.081774   1.6594216]]`

**Comment by BonyJordan** [comment]
I’d have to see the line of code that produced it to be sure, but possibly those are softmax logits for the outcomes [I win, draw, they win]
